src/Root.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './pages/PeoplePage';
+import { NotFound } from './pages/NotFound';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="home" element={<Navigate to="/" replace />} />
+        <Route path="people">
+          <Route index element={<PeoplePage />} />
+          <Route index path=":slug" element={<PeoplePage />} />
```

Comment:
There is a redundancy in the `people` route configuration. The `index` attribute is used twice: once for the default `PeoplePage` and again with a `path` attribute. Consider removing the `index` from the second route or adjusting the configuration to avoid confusion.